### ISSUE TITLE
Color scheme

### ISSUE THREAD

**Issue by Oryhara** [djscheuf/journeymans-blog#145]
I don't like this either. That is the current color scheme. background is taupe(?) which is a hideous color. I feel like i'm reading something written on papyrus. Papyrus is old. 
Don't be old. 

the psychology of color is a science and art unto itself, but a quick splash in that deep, deep pool of knowledge would suggest the use of:

Blue: for trust, dependable, strength
Purple: for Creativity, Imagination, and wisdom

**Comment by djscheuf** [comment]
Indeed, The _old_ color scheme _was_ on a papyrus color, which _was_ intentional at the time. But over time I realized it wasn't sending the right message. Are you still seeing that old scheme on the test site @Oryhara ?

**Comment by Oryhara** [comment]
Screenshots of every page listed in the first email soliciting feedback. screenshot of a browser window 1920x1080. 
please excuse my bookmarks bar i didn't think of hiding them for the screenshots. 

subsequent email indicates main color scheme should be #f6f4ff and it is not.

imho even #f6f4ff is too subtle. I'd suggest more blue. but maybe i'm bias. also get some bold headers/borders like on that fsharp site. 

I'll take #0000ff any day of the week but I didn't grow up with windows so it doesn't trigger BSOD flashbacks. ymmv


![Screenshot 2022-07-26 104118](https://user-images.githubusercontent.com/1857202/181050713-0a64cc8b-43e7-451b-83b4-3738082e904f.png)
![Screenshot 2022-07-26 104133](https://user-images.githubusercontent.com/1857202/181050715-9fc07f32-f4d6-4b0e-b111-3b4c1f982a1e.png)
![Screenshot 2022-07-26 104151](https://user-images.githubusercontent.com/1857202/181050716-fe8af291-b007-4dde-9e77-a95b8eb19763.png)
![Screenshot 2022-07-26 104203](https://user-images.githubusercontent.com/1857202/181050717-3709330c-1eb3-4831-a1f0-4c2b6b5a3308.png)
![Screenshot 2022-07-26 104212](https://user-images.githubusercontent.com/1857202/181050718-9038bda9-ac23-41b7-8104-bb8faef5e18a.png)
![Screenshot 2022-07-26 104221](https://user-images.githubusercontent.com/1857202/181050719-0654363c-7592-4692-8bc2-df46297d9833.png)
![Screenshot 2022-07-26 104231](https://user-images.githubusercontent.com/1857202/181050720-62ad31ee-db52-49eb-aa1f-91b125aeae80.png)
![Screenshot 2022-07-26 104238](https://user-images.githubusercontent.com/1857202/181050722-f39587db-5ed4-43a2-9a15-6b5348c03a21.png)

**Comment by djscheuf** [comment]
Issue closed, as the previous color scheme has since been replaced. And the Papyrus background color updated.
